honeycomb/honeycomb_test.go

```diff
@@ -123,28 +123,40 @@ func TestExport(t *testing.T) {
 	}
 }
 
-func setUpTestExporter(mockHoneycomb *libhoney.MockOutput, opts ...ExporterOption) (apitrace.Tracer, error) {
-	exporter, err := NewExporter(
+func makeTestExporter(mockHoneycomb *libhoney.MockOutput, opts ...ExporterOption) (*Exporter, error) {
+	return NewExporter(
 		Config{
 			APIKey: "overridden",
 		},
 		append(opts,
 			TargetingDataset("test"),
 			WithServiceName("opentelemetry-test"),
-			withHoneycombOutput(mockHoneycomb))...)
+			withHoneycombOutput(mockHoneycomb))...,
+	)
+}
+
+func setUpTestProvider(exporter exporttrace.SpanSyncer, opts ...sdktrace.ProviderOption) (apitrace.Tracer, error) {
+	tp, err := sdktrace.NewProvider(
+		append([]sdktrace.ProviderOption{
+			sdktrace.WithConfig(sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}),
+			sdktrace.WithSyncer(exporter),
+		},
+			opts...)...,
```

Comment:
I'm surprised this is what gofmt wants, but as long as gofmt says it's okay...

Reply:
Me too! I tried reformatting this several times, and couldn't bend it to what I expected it to look like.

Would you prefer a different wrapping? Perhaps `opts...` after the comma above?

Reply:
yeah, I think after the comma above would read more cleanly.

Reply:
Hey, _go fmt_ acceded. I amended that commit.